dev-requirements.txt

```diff
@@ -26,7 +26,7 @@ matplotlib>=2.2.3
 aiohttp
 
 # Required to run plugin tests.
-pip>=20.0.1
+pip
```

Comment:
Is it relevant to have this here at all anymore, as pip will certainly be installed in a user's environment?

Reply:
I'll remove it. It could happen in Docker or Conda for example, but if they use this file, they likely have it installed.